planner/core/resolve_indices.go

```diff
@@ -258,6 +267,15 @@ func (p *PhysicalIndexReader) ResolveIndices() (err error) {
 
 // ResolveIndices implements Plan interface.
 func (p *PhysicalIndexLookUpReader) ResolveIndices() (err error) {
+	for _, col := range p.tablePlan.Schema().Columns {
```

Comment:
Could you extract a function for them? It is duplicated with physical table reader.

Reply:
Done！